src/About/SupportCard.tsx

```diff
@@ -83,7 +87,7 @@ export default () => {
                         <Button
                             variant="secondary"
                             onClick={() => {
-                                persistIsLoggingVerbose(true);
+                                removeIsLoggingVerboseResetHandler();
```

Comment:
W: Instead of expressing the “how” with `removeIsLoggingVerboseResetHandler` I would name this to more express the “what/why”, e.g. with `doNotResetLoggingOnRestart`.